system/security/securesocket/securesocket.cpp

```diff
@@ -1999,9 +2020,16 @@ class CSecureSmartSocketFactory : public CSmartSocketFactory
 
 ISmartSocketFactory *createSecureSmartSocketFactory(const char *_socklist, bool _retry, unsigned _retryInterval, unsigned _dnsInterval)
 {
+    DBGLOG("createing smart socket: TLS, %s", _socklist);
     return new CSecureSmartSocketFactory(_socklist, _retry, _retryInterval, _dnsInterval);
 }
 
+ISmartSocketFactory *createSecureSmartSocketFactory(IPropertyTree &service, const char *defPort, bool _retry, unsigned _retryInterval, unsigned _dnsInterval)
+{
+    DBGLOG("createing service smart socket: TLS");
```

Comment:
Could this tracing become excessive?  Same for the function above.  (Also typo in creat**e**ing )